### PR TITLE
fix: Cannot resolve `model_card` GQL query

### PR DESCRIPTION
**Pull request by fregataa**
follow-up #1749 

**Checklist:** (if applicable)

- [x] Milestone metadata specifying the target backport version
- [x] API server-client counterparts (e.g., manager API -> client SDK)

### CODE REVIEW

**Review by yomybaby — APPROVED** [review]
LGTM

File: src/ai/backend/manager/models/vfolder.py [hunk 1]
```diff
@@ -2222,6 +2251,7 @@ class Meta:
 
     name = graphene.String()
     vfolder = graphene.Field(VirtualFolder)
+    vfolder_node = graphene.Field(VirtualFolderNode, description="Added in 24.09.0.")
```
Comment (yomybaby) [inline src/ai/backend/manager/models/vfolder.py:2254]: If this PR will also be merged into ~~24.09.x~~ 24.03.x, please update this description.

**Review by yomybaby — CHANGES_REQUESTED** [review]

File: src/ai/backend/manager/models/vfolder.py (hunk 1 above)
Comment (fregataa) [reply src/ai/backend/manager/models/vfolder.py:2254]: As I know, `24.09.0` is the first minor version of 24.09. Does any other version come to prior to 24.09.0?

Comment (fregataa) [reply src/ai/backend/manager/models/vfolder.py:2254]: As we confirmed that the target milestone is `24.09`, this conversation has been resolved.

**Review by yomybaby — CHANGES_REQUESTED** [review]
LGTM

**Review by yomybaby — APPROVED** [review]
LGTM

**Pull request merged**